### ISSUE TITLE
Why not collect some loss functions?

### ISSUE THREAD

**Issue by cy69855522** [Cadene/pretrained-models.pytorch#85]
I don't know what to say to thank you for this project, it helps a lot of people, and I would like to suggest, if you can, collecting some loss functions will make it more perfect!
